core/oslib/audiobackend_coreaudio.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+    Simple Core Audio backend for osx (and maybe ios?)
+    Based off various audio core samples and dolphin's code
+ 
+    This is part of the Reicast project, please consult the
+    LICENSE file for licensing & related information
+ 
+    This could do with some locking logic to avoid
+    race conditions, and some variable length buffer
+    logic to support chunk sizes other than 512 bytes
+ 
+    It does work on my macmini though
+ */
+
+#include "oslib/audiobackend_coreaudio.h"
+
+#if HOST_OS == OS_DARWIN
+
+//#include <CoreAudio/CoreAudio.h>
+#include <AudioUnit/AudioUnit.h>
+
+AudioUnit audioUnit;
```

Comment:
Maybe, but I generally don't use the "static" keyword cus I don't like how it sounds/looks/feels. Linkage control has nothing to do with "staticity". So I add it only when it becomes a problem...